### PR TITLE
TIDoS-Framework: initial commit

### LINKED ISSUE #2911: TIDoS-Framework
- **Name**: TIDoS-Framework
- **Source URL**: https://github.com/0xInfection/TIDoS-Framework
- Recon framework
- Asked by Djordi#7904 on discord

### PR DESCRIPTION
**Pull request by D4imyo**
This PR fixes #2911

check tidos-framework depends variable.

### CODE REVIEW

File: packages/python-mysql/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-mysql
```
Comment (noraj) [inline packages/python-mysql/PKGBUILD:4]: > This package is a ‘virtual package’, which requires MySQL-python (Python 2) or mysqlclient (Python 3) to install. In effect, this means ‘pip install mysql’ will actually install MySQL-python.

```
$ pacman -Ss python-mysql                                  
community/python-mysql-connector 8.0.22-2
    A standardized database driver for Python platforms and development
community/python-mysqlclient 2.0.3-1
    Fork of MySQL-python with support for Python 3
```

So don't create this package and only require `python-mysqlclient` instead.

**Review by noraj — CHANGES_REQUESTED** [review]

**Review by noptrix — APPROVED** [review]
@D4imyo @noraj thanks a lot! can we merge in? :) looks good to me

**Comment by noraj** [comment]
LGTM but did not tested

**Comment by noptrix** [comment]
> LGTM but did not tested

thx! will check it later and merge in:)

**Pull request merged**